borg/testsuite/archiver.py

```diff
@@ -71,6 +71,29 @@ def __exit__(self, *args, **kw):
                 os.environ[k] = v
 
 
+def exec_cmd(*args, archiver=None, fork=False, **kw):
```

Comment:
well then why isn't the static method used everywhere in the tests?

or otherwise, this at least needs some justification in docstrings, imho.